src/components/BookList.js

```diff
@@ -1,24 +1,17 @@
-import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
 import BookItem from './BookItem';
 
-const BookList = ({ booksProps }) => (
-  <ul>
-    {
-                booksProps.map((booksArray) => (
-                  <BookItem key={booksArray.id} bookListProp={booksArray} />
-                ))
-            }
-  </ul>
-);
-
-BookList.propTypes = {
-  booksProps: PropTypes.arrayOf(
-    PropTypes.shape({
-      id: PropTypes.number.isRequired,
-      title: PropTypes.string.isRequired,
-      author: PropTypes.string.isRequired,
-    }),
-  ).isRequired,
+const BookList = () => {
+  const { books } = useSelector((state) => state.books);
+  return (
+    <ul>
+      {
+        books.map((book) => (
```

Comment:
- [ ] Great job on implementing the milestone how ever please i would kindly suggest you to Make sure that useEffect is used when dispatching the fetchBooks function in <BookList>as this will Refactor your add book and remove book features to persist your changes in the server
````jsx
import { useDispatch, useSelector } from "react-redux"
import { useEffect } from "react"
import { fetchBooks } from "./store/books/booksSlice"

function BookList() {
  const dispatch = useDispatch()
  const books = useSelector((state) => state.books.books)

  useEffect(() => {
    dispatch(fetchBooks())
  }, [])

  return <div>{/* BookList JSX including books data */}</div>
}

export default App
````